pkgs/by-name/sd/SDL2_image_2_0/package.nix

```diff
@@ -1,17 +0,0 @@
-# Dependency of pygame, toppler
```

Comment:
Thanks for dropping this! This should however get an entry in `pkgs/top-level/aliases.nix`